core/src/main/java/com/amplifyframework/hub/AWSHubPlugin.java

```diff
@@ -116,22 +121,25 @@ public void unsubscribe(@NonNull SubscriptionToken subscriptionToken) {
 
             // Now that we have a handle to the subscription, figure out which channel
             final HubChannel channelToUpdate = subscriptionBeingEnded.getHubChannel();
-            if (subscriptionsByChannel.containsKey(channelToUpdate)) {
-                subscriptionsByChannel.get(channelToUpdate).remove(subscriptionBeingEnded);
+            final Set<HubSubscription> channelSubscriptions = subscriptionsByChannel.get(channelToUpdate);
+            if (channelSubscriptions != null) {
+                channelSubscriptions.remove(subscriptionBeingEnded);
             }
         }
     }
 
+    @NonNull
     @Override
     public String getPluginKey() {
         return "awsHubPlugin";
     }
 
     @Override
-    public void configure(@NonNull JSONObject pluginConfiguration, Context context) {
+    public void configure(@NonNull JSONObject pluginConfiguration, @NonNull Context context) {
 
     }
 
+    @NonNull
     @Override
     public Void getEscapeHatch() {
         return null;
```

Comment:
hmmmmmm

Reply:
Ah yes, this should say `@Nullable`, will update